__tests__/app.tests.js

```diff
@@ -445,4 +445,45 @@ describe('POST /api/articles/:article_id/comments', () => {
       })
     })
   })
-  })
\ No newline at end of file
+  })
+
+  describe('comment count addition', () => {
+    test('Status 200: responds with a comment count for article 2 (cooment count exists)', () => {
+      return request(app)
+      .get('/api/articles/1?comment_count=true')
+      .expect(200)
+      .then((response) => {
+        const article = response.body.article;
+        expect(article).toMatchObject({
+          article_id: 1,
+          title: "Living in the shadow of a great man",
+          topic: "mitch",     
+          author: "butter_bridge",
+          body: "I find this existence challenging",
+          created_at: expect.any(String),
+          votes: 100,
+          comment_count: "11"
+        })
+      })
+    })
```

Comment:
Minor misunderstanding of the endpoint here Hannah - we didn't want you to add a query to the endpoint, rather to implement the behaviour to the endpoint as is - however, this is your project and I like it being a query so it's up to you 👍